tests/integration/opcache/disabled/skipif.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Opcache only supported with PHP 7+ */
+if (version_compare(PHP_VERSION, "7.1", "<")) {
+  die("skip: PHP < 7.1 not supported\n");
+}
+
+/* Test should not run with opache */
+if (extension_loaded('Zend OPcache')) {
+  die("warn: Zend OPcache extension required to be not loaded\n");
```

Comment:
So if integration_runner always loads opcache, won't extension_loaded('Zend OPcache') always return true and therefore none of the tests with this check will run?

Reply:
I don't think this check is needed at all - tests should run regardless if opcache is loaded or not. If it's loaded, it will be disabled with:
```
/*INI
opcache.enable=0
opcache.enable_cli=0
```
If it is not loaded, then opcache will be disabled as well.

Reply:
I think that I would rather this check be a failure case within each test here. The tests expect opcache to be disabled and set the INI values as such. Therefore, failing this check should fail the test, not skip it. @lavarou do you agree?

Reply:
changed in [367b613](https://github.com/newrelic/newrelic-php-agent/pull/752/commits/367b6137b52604294ec32ff8df8f3fc06818d2f6)